tools/nessus/scan/Modules/Admin.txt

```diff
@@ -0,0 +1,2 @@
+# Check if user is part of the administrator group
```

Comment:
Maybe remove admin and firewall script as these are meant to be added in another PR anyway and they have been added to a backup branch earlier.